### ISSUE TITLE
Lost list spells and order validation/checking

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JOverseer/joverseer#504]
```
This one's a small problem, as it doesn't actually stop me issuing the
order; it just gives me a false error message when I run the order checker.

Specifically, I took SNA #25, which allows my mages to learn the Conjure
Food spell. Because this is a "Lost List" spell, researching it doesn't
work the same way as spells from "normal" lists.

The SNA is the one and only pre-requisite for learning the spell, even
though it's an "Average" difficulty spell, and the mage hasn't learned the
"Easy" Conjure Mounts spell from the same list.

When I run the Order Checker, it red flags my "705 RsrchSp 510" order, and
says my mage lacks the prerequisite. But I cannot ever have a mage learn
Conjure Mounts, because I didn't take that SNA, only the Conjure Food one.
However, if I do "705 RsrchSp 508", it merely yellow flags the order, and
tells me it is on a Lost List, then green flags it, because the mage "has
the prerequisite". It does the same if I give order "705 RsrchSp 512" to a
mage who already knows Conjure Food, or have a mage with Conjure Mounts try
to learn Conjure Food or Conjure Hordes.

All of these are wrong. I didn't take SNA #24, so I can't have a mage
research Conjure Mounts. Likewise, my other mage, who started with Conjure
Food, can never learn Conjure Hordes, even though he "has the prerequisite"
for it.

In short, on 705 orders for "Lost List" spells, the only thing the checker
should look at is whether my nation has the SNA that allows my mages to
learn the spell, not whether or which other spells on the list I know. If I
have the SNA, my mage can learn the spell (and he did). If I don't have the
SNA, he can't.

```

Original issue reported on code.google.com by `mscoon` on 25 Oct 2009 at 9:12

**Comment by EdwardLaneUK** [comment]
understood, should be able to fix that
